ingest/pipeline/register.go

```diff
@@ -0,0 +1,69 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package pipeline
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/elastic/beats/libbeat/logp"
+	es "github.com/elastic/beats/libbeat/outputs/elasticsearch"
+)
+
+func RegisterPipelines(esClient *es.Client, overwrite bool, path string) error {
+	pipelines, err := loadPipelinesFromJSON(path)
```

Comment:
is this path being resolved correctly with deb/yum/etc packaging?

Reply:
ok, i saw now that you are packing the json file with mage 👍 
do you want me to test that or are you confident with it?

Reply:
I tested it only with the tar package so far, but can do more testing.

Reply:
i think eg linux packaging is interesting because assets would end in another directory`/etc/something/something` i presume? 
lmk if you want any help testing that

Reply:
sure, i can test that 👍